src/yuzu/util/util.cpp

```diff
@@ -42,7 +50,27 @@ QPixmap CreateCirclePixmapFromColor(const QColor& color) {
     return circle_pixmap;
 }
 
-bool SaveIconToFile(const std::string_view path, const QImage& image) {
+QString GetTargetPath(GameListShortcutTarget target) {
```

Comment:
```suggestion
QString GetShortcutPath(GameListShortcutTarget target) {
```

Sounds way less generic.